### ISSUE TITLE
Version 4.65 introduced a breaking change for plugins

### ISSUE THREAD

**Issue by Archomeda** [JosefNemec/Playnite#922]
Version 4.65 introduced the property `Playnite.SDK.Plugins.ILibraryPlugin.IsClientInstalled`, which is a breaking change for existing plugins. .NET can no longer create instances of plugins because the interface is different from version 4.64 and SDK 3.0.

Basically, when running Playnite with `ThrowAllErrors` set to `True`, it will throw an exception with the message that the method get_IsClientInstalled is not implemented in the plugin.
If you're running Playnite with `ThrowAllErrors` set to `False`, it will only log the message that the library failed to load, which isn't helpful (better logging would be appreciated in case a plugin failed to load).

In order to fix this without waiting for an SDK update (or a revert with backwards compatibility), you have to add your own interface and implement it this way:

```cs
using Playnite.SDK.Plugins;

namespace MyPlugin
{
    public interface ILibraryPlugin465 : ILibraryPlugin
    {
        bool IsClientInstalled { get; }
    }

    public class MyLibraryPlugin : ILibraryPlugin465
    {
        // Implementation...
    }
}
```

Directly implementing the property without the interface doesn't seem to work in my case.

**Comment by JosefNemec** [comment]
I totally forgot that we added that property. I'll update SDK nuget and push new program update early tomorrow.

**Comment by JosefNemec** [comment]
Fixed in 4.66 and new nuget 4.0.0.
